doc/manualGrading.md

```diff
@@ -0,0 +1,56 @@
+# Manual Grading
+
+PrairieLearn supports manual grading of questions by downloading a CSV file with student answers and uploading a CSV file with question scores and optional per-question feedback. There is not currently an online web interface for manual grading.
+
+
+## Configuring a question for maual grading
+
+The [`info.json` file](question.md#question-infojson) in the question should be set for manual grading:
+```json
+{
+    "uuid": "cbf5cbf2-6458-4f13-a418-aa4d2b1093ff",
+    "gradingMethod": "Manual",
+    "singleVariant": true,
+    ...
+}
+```
+
+Manually-graded questions allow students to "Save" answers, but they don't have a "Save & Grade" button. Instead, the student just saves answers as many times as they want, and all of their submitted answers are stored. It is recommended to also mark manually-graded questions as `"singleVariant": true` so that students are only given a single random variant, even on Homework assessments.
+
+Any [elements](elements/) can be used in the [`question.html`](question.md#question-questionhtml) to write manually graded questions. All of the student input will be saved and available for manual grading, including `pl-string-input`, `pl-file-editor`, `pl-file-upload`, etc.
+
+To show manual feedback the `question.html` file should contain an element to display the feedback next to student submissions. A basic template for this is:
+```html
+<pl-submission-panel>
```

Comment:
Any reason for not recommending it be placed in `pl-answer-panel`?

Reply:
Feedback is specially associated with a submission, because it's a comment on the submission itself. The "answer" panel is supposed to be for the "gold standard answer".